### ISSUE TITLE
throw bug in PSO

### ISSUE THREAD

**Issue by Sceki** [esa/pagmo2#322]
In src/pso.cpp, the lines:
```
    if (m_omega < 0. || m_omega > 1.) {
        // variants using Inertia weight
        pagmo_throw(std::invalid_argument,
                    "The particles' inertia (or the constriction factor) must be in the [0,1] range, while a value of "
                        + std::to_string(m_variant) + " was detected");
    }
```

Should be :
```
    if (m_omega < 0. || m_omega > 1.) {
        // variants using Inertia weight
        pagmo_throw(std::invalid_argument,
                    "The particles' inertia (or the constriction factor) must be in the [0,1] range, while a value of "
                        + std::to_string(m_omega) + " was detected");
    }
```

`m_omega`, instead of `m_variant` should thus be used in the throw message.

**Comment by Sceki** [comment]
I am doing a PR for this..

**Comment by darioizzo** [comment]
:) thanks!

**Comment by bluescarni** [comment]
Fixed by #323.
